### PR TITLE
Connect update images

### PR DESCRIPTION
**Pull request by ZhraaBaharAluloom**

### CODE REVIEW

File: src/components/EditImageModal.jsx [hunk 1]
```diff
@@ -1,23 +1,74 @@
...
+import { db } from "../firebase";
+import { getDownloadURL, ref, uploadBytes } from "@firebase/storage";
+import { storage } from "../firebase";
+import Loader from "./Loader";
 const el = document.getElementById("root");
 Modal.setAppElement(el);
 
-function EditImageModal({ isEditImageOpen, setIsEditImageOpen }) {
+function EditImageModal({ isEditImageOpen, setIsEditImageOpen, book }) {
+  // const [images, setImages] = useState([]);
```
Comment (MuhannedNoman) [inline src/components/EditImageModal.jsx]: Remove this if not needed

File: src/pages/Books.jsx [hunk 2]
```diff
@@ -31,6 +32,9 @@ function Books() {
   const indexOfLastPost = currentPage * postsPerPage;
 
   const indexOfFirstPost = indexOfLastPost - postsPerPage;
+
+  // const currentPosts = posts?.slice(indexOfFirstPost, indexOfLastPost);
+  // const howManyPages = Math.ceil(books.length / postsPerPage);
```
Comment (MuhannedNoman) [inline src/pages/Books.jsx]: Remove comments if not needed.

**Review by MuhannedNoman — APPROVED** [review]
LGTM  just left a few comments.

File: src/pages/Books.jsx (hunk 2 above)
Comment (ZhraaBaharAluloom) [reply src/pages/Books.jsx]: done

File: src/components/EditImageModal.jsx (hunk 1 above)
Comment (ZhraaBaharAluloom) [reply src/components/EditImageModal.jsx]: done

**Pull request merged**